core/services/chainlink/application.go

```diff
@@ -701,3 +702,35 @@ func (app *ChainlinkApplication) AddServiceAgreement(sa *models.ServiceAgreement
 func (app *ChainlinkApplication) NewBox() packr.Box {
 	return packr.NewBox("../../../operator_ui/dist")
 }
+
+// Returns the configuration to use for creating and authenticating
+// new WebAuthn credentials
+func (app *ChainlinkApplication) GetWebAuthnConfiguration() models.WebAuthnConfiguration {
+	if os.Getenv("RPID") == "" {
```

Comment:
Generally it's preferable to use the `Config` object rather than reading from env vars directly

Reply:
So you can add a field to schema.go, a method in config.go then read it here as `app.Config.RPID()` for example.